scripts/import/import_all_sv_data.pl

```diff
@@ -420,15 +431,15 @@ sub study_table{
     elsif ($assembly eq $target_assembly && $study_desc =~ /^(.+)\s\[remapped/) {
       $study_desc = $1;
     }
-    
+
     my $external_link_sql;
     if ($external_link eq 'NULL') {
-      if ($study_xref && $study_xref !~ /NULL/i && $study_xref ne '') {
+      # if ($study_xref && $study_xref !~ /NULL/i && $study_xref ne '') {
         $external_link_sql = '';
-      }
-      else {
-        $external_link_sql = "external_reference='$external_link',";
-      }
+      # }
```

Comment:
Is there a reason why we keep this as a comment?

Reply:
This is the same situation as the previous comment. Shouldn't be commented, I forgot to fix it before PR.